src/aof.c

```diff
@@ -89,6 +94,22 @@ unsigned long aofRewriteBufferSize(void) {
     return size;
 }
 
+/* This function is different from aofRewriteBufferSize, to get memory usage,
+ * we should also count all other fields(except 'buf') of aofrwblock and the
+ * last block's free size. */
+unsigned long aofRewriteBufferMemoryUsage(void) {
```

Comment:
hi @oranagra here, i want to say, if we want to get aof rewrite buffer more accurate, we maybe use `zmalloc_size` to get every `aof_rewrite_buf_block`, but as you said, it costs much time, so I actually cumulate all `aof_rewrite_buf_blocks` zmalloc_usable_size, of course, they(zmalloc_size and zmalloc_usable_size) have no difference if allocator has MALLOC SIZE.